### ISSUE TITLE
Ordering of project tabs

### ISSUE THREAD

**Issue by poppa** [poppa/resourceful#2]
The project tabs are now ordered by the order of the project directories, which are UUIDs, so the ordering is pretty much random, albeit consistent between restarts.

The tab order should be saved to disk, and the tabs should also be re-orderable.
